AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/core/CustomisedGsonBuilder.java

```diff
@@ -18,8 +16,10 @@
 
     public static Gson getGson() {
         return new GsonBuilder()
+
                 .registerTypeAdapter(LocalDate.class, new LocalDateAdapter())
                 .registerTypeAdapter(ZonedDateTime.class, new ZonedDateTimeAdapter())
+                .serializeNulls()
```

Comment:
This looks like it shouldn't be in this review? Unless you also needed it for long/lat?